plugins/modules/redis_info.py

```diff
@@ -2,49 +2,69 @@
 # -*- coding: utf-8 -*-
 
 # Copyright (c) 2020, Pavlo Bashynskyi (@levonet) <levonet@gmail.com>
-# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# Copyright (c) 2023, Xcelirate (@vicmunoz) <victor1 at teamxcl dot com>
+# GNU General Public License v3.0+
+# (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
 
-DOCUMENTATION = r'''
+DOCUMENTATION = '''
 ---
 module: redis_info
 short_description: Gather information about Redis servers
 version_added: '0.2.0'
 description:
-- Gathers information and statistics about Redis servers.
+    - Gathers information and statistics about Redis servers.
 extends_documentation_fragment:
-- community.general.attributes
-- community.general.attributes.info_module
+    - community.general.redis
+    - community.general.attributes
+    - community.general.attributes.info_module
 options:
-  login_host:
-    description:
-    - The host running the database.
-    type: str
-    default: localhost
-  login_port:
-    description:
-    - The port to connect to.
-    type: int
-    default: 6379
-  login_password:
-    description:
-    - The password used to authenticate with, when authentication is enabled for the Redis server.
-    type: str
+    login_host:
+        description:
+            - The host running the database.
+        type: str
+        default: localhost
+    login_port:
+        description:
+            - The port to connect to.
+        type: int
+        default: 6379
+    login_password:
+        description:
+            - The password used to authenticate with, when authentication is enabled for the Redis server.
+        type: str
```

Comment:
These are provided by the docs fragment, there's no need to replicate them (with slightly different descriptions).
```suggestion
```